packages/client/wallets/aa/front/components/CreateWallet.tsx

```diff
@@ -9,7 +9,7 @@ interface CreateWalletComponentProps {
     onWalletCreated: (wallet: EVMAAWallet) => void;
 }
 
-const CreateWalletComponent: React.FC<CreateWalletComponentProps> = ({ onWalletCreated }) => {
+function CreateWalletComponent({ onWalletCreated }: CreateWalletComponentProps) {
```

Comment:
`React.FC` is not the proper way to define a react component anymore, so when updating some versions these components must be updated as well